packages/pytest-simcore/src/pytest_simcore/redis_service.py

```diff
@@ -93,5 +93,9 @@ async def redis_locks_client(
 async def wait_till_redis_responsive(redis_url: Union[URL, str]) -> None:
     client = from_url(f"{redis_url}", encoding="utf-8", decode_responses=True)
 
-    if not await client.ping():
-        raise ConnectionError(f"{redis_url=} not available")
+    try:
```

Comment:
Why do you need to flushall here? it is just a ping...
would it not make more sense to separate all this, or even use the sync client in that case? just in a context manager...

Reply:
No need to flushall, you are correct. The issue is with this `close_connection_pool` argument which is not supplied when using the context manger.